### DISCUSSION TITLE
remote repository

### DISCUSSION THREAD

**Discussion by vathenao** [AcademySoftwareFoundation/rez#1440] [Q&A] [answered]
can rez support remote repository?  i hope it can works like other packages management tools do, find the packages on remote repository(on my server typically) but not a share disk.

**Comment by JeanChristopheMorinPerso** [comment, marked as answer]
Hi @vathenao ! Unfortunately, it's not something that can be done out of the box. But it's possible to do using plugins. For example someone created a PR for storing artifacts in S3, see #1380.

What's your use case? Is it because using a file share is too slow, or because you want to use Rez un containers, or in cloud environments, or something else?

**Reply by vathenao** [reply]
We provide a client to help using autodesk maya in a specific environment and rez good at ding this. But, because of we publish packages on share disk, so other users must mount a shared disk during install our client in order to use newest packages. And this make the installation is to slow. We hope our client is more and more light-weight.

**Reply by vathenao** [reply]
And Just like you said, initialize an environment from share disk is too slow.

**Reply by instinct-vfx** [reply]
Not saying there is not a valid case to be made for remote repositories - but for the time being have you tried rez's payload caching system? We are using this very successfully here.

**Reply by vathenao** [reply]
yeat, after enable caching system, the startup becomes faster. But we think that mount a shared disk when install our client is a poor experience.

**Reply by JeanChristopheMorinPerso** [reply]
We understand that and it is on our radar. It is a commonly requested feature and it's also important to the TSC members.

You do have an escape hatch in the PR in linked of you really want to get rid of your network storage.
